mobile/styles/line_styles.js

```diff
@@ -0,0 +1,29 @@
+import { StyleSheet } from "react-native";
+
+export const lineStyle = StyleSheet.create({
+    line: {
+        backgroundColor: "#FFF5D2",
+        width: "85%",
+        height: 6,
+        position: "relative",
+        marginBottom: 25,
```

Comment:
Ik denk dat deze margin te groot is. want er is een te grote ruimte tussen de header en de route overzicht. Maar kan ook zijn dat er ergens anders een margin te groot is.